task11/3.sql

```diff
@@ -0,0 +1,3 @@
+use mydb;
+select * from posts
+where idOfUser = "6" and position("hello" in description) <> 0;
```

Comment:
"position" returns the position of substring in string and numeration of chars is from 1.
But this request won't work because I forgot to fix "idOfUser" on "username"